backend/iam/adapter.py

```
@@ -10,13 +10,16 @@
 from django.conf import settings
 from django.contrib.auth import get_user_model
 from django.db.models import Q
+from django.http import HttpResponseRedirect
```

Ruff (0.11.9)

`django.http.HttpResponseRedirect` imported but unused

Remove unused import: `django.http.HttpResponseRedirect`

(F401)